app/src/app/dashboard/dashboard.component.ts

```diff
@@ -221,18 +227,38 @@ export class DashboardComponent implements OnInit, OnDestroy {
       });
   }
 
+  private onNewestPageChange(event) {
+    this.newestPage = event;
+    this.loadNewestObjects();
+  }
+
+  private  onLatestPageChange(event) {
+    this.latestPage = event;
+    this.loadLatestObjects();
+  }
+
   private loadNewestObjects(): void {
-    this.objectService.getNewestObjects().pipe(takeUntil(this.unSubscribe)).subscribe((apiResponse: APIGetMultiResponse<RenderResult>) => {
+    this.newestLoading = true;
+    const apiParameters: CollectionParameters = {page: this.newestPage, limit: 10, order: -1};
```

Comment:
It would have been simpler to just pass the parameters in the function, so you wouldn´t have the global variables. Is less error-prone. But this is only a small inefficiency.